homeassistant/components/nibe_heatpump/const.py

```diff
@@ -17,3 +17,4 @@
 VALUES_MIXING_VALVE_CLOSED_STATE = (30, "CLOSED", "SHUNT CLOSED")
 VALUES_PRIORITY_HEATING = (30, "HEAT")
 VALUES_PRIORITY_COOLING = (60, "COOLING")
+VALUES_COOL_WITH_ROOM_SENSOR_OFF = (0, "OFF")
```

Comment:
In your PR description you refer to `"0"` (string of the number 0). Is the code or the PR description correct?

Reply:
Description. It was a way to highlight the number.